.github/workflows/release-executables.yml

```diff
@@ -107,46 +107,17 @@ jobs:
           mv ./dist/Noco-macos-arm64 ./mac-dist/
           mv ./dist/Noco-macos-x64 ./mac-dist/
 
-      - name: Upload win-arm64 build to asset
+      - name: Upload executables to asset
         id: upload-release-asset
-        uses: actions/upload-release-asset@v1
+        uses: softprops/action-gh-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ./scripts/pkg-executable/dist/Noco-win-arm64.exe
-          asset_name: Noco-win-arm64.exe
-          asset_content_type: application/octet-stream
-
-      - name: Upload win-x64 build to asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ./scripts/pkg-executable/dist/Noco-win-x64.exe
-          asset_name: Noco-win-x64.exe
-          asset_content_type: application/octet-stream
-
-      - name: Upload linux-arm64 build to asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ./scripts/pkg-executable/dist/Noco-linux-arm64
-          asset_name: Noco-linux-arm64
-          asset_content_type: application/octet-stream
-
-      - name: Upload linux-x64 build to asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ./scripts/pkg-executable/dist/Noco-linux-x64
-          asset_name: Noco-linux-x64
-          asset_content_type: application/octet-stream
+          files: |
+             ./scripts/pkg-executable/dist/Noco-win-arm64.exe
+             ./scripts/pkg-executable/dist/Noco-win-x64.exe
+             ./scripts/pkg-executable/dist/Noco-linux-arm64
+             ./scripts/pkg-executable/dist/Noco-linux-x64
```

Comment:
Do we need `get_release` step now? ( L25 - L31 )

Reply:
confirmed that it is no longer required